examples/file-sharing.rs

```diff
@@ -134,8 +135,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
                     // Reply with the content of the file on incoming requests.
                     Some(network::Event::InboundRequest { request, channel }) => {
                         if request == name {
-                            let file_content = std::fs::read_to_string(&path)?;
-                            network_client.respond_file(file_content, channel).await;
+                            network_client.respond_file(&std::fs::read(&path)?, channel).await;
```

Comment:
```suggestion
                            network_client.respond_file(std::fs::read(&path)?, channel).await;
```

Why borrow here when we clone again later on?

Reply:
but expected `&Vec<u8>`  here.  any better choice?